analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -509,7 +518,18 @@ def add_arguments_to_parser(parser):
                                     "OWN RISK!")
 
     logger.add_verbose_arguments(parser)
-    parser.set_defaults(func=main)
+    parser.set_defaults(func=main,
+                        func_process_config_file=process_config_file)
+
+
+def process_config_file(args):
+    """
+    Handler to get config file options.
+    """
+    if 'config' in args and os.path.exists(args.config):
+        with open(args.config) as config_file:
```

Comment:
I would highly recommend to use some well know config file format (json, [python config](https://docs.python.org/2/library/configparser.html)) it will be much easier to extend or process it later.